lib/Twig/Extension/Core.php

```
@@ -852,6 +852,34 @@ function twig_in_filter($value, $compare)
     return false;
 }
 
+/** 
+ * Returns a trimmed string.
+ * 
+ * @param string $string
+ * @param string $characterMask
+ * @param string $mode
+ * @return string
+ */
+function twig_trim_filter($string, $characterMask = null, $mode = 'both')
+{
+    if (empty($characterMask)) {
+        $characterMask = " \t\n\r\0\x0B";
```

Understood. How should we handle this first parameter when people want to have the second parameter but use only the default mask?